internal/migration_acceptance_tests/sequence_cases_test.go

```diff
@@ -640,6 +598,74 @@ var sequenceAcceptanceTests = []acceptanceTestCase{
 			diff.MigrationHazardTypeDeletesData,
 		},
 	},
+	{
```

Comment:
It might make sense to include some test cases with `NO MINVALUE` and `NO MAXVALUE`, particularly when there's a type change? I'm not really sure what the behavior _should_ be in that scenario since the defaults would change...

Reply:
Solid points. Added some test cases for those scenarios.